content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,112 @@
+===============================================================
+Use Windows IoT software to connect devices to an Odoo database
+===============================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
+computer. This requires a Windows operating system with an Odoo 16 or later database.
+
+The Windows Virtual :abbr:`IoT (Internet of Things)` box works the same way as a physical
+:abbr:`IoT (Internet of Things)` box, with the ability to run most of the same devices. All
+:abbr:`POS (Point of Sale)` devices work with it, such as a scale or printer. Payment terminals will
+also work, but it should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not
+compatible. These include cameras or measurement tools.
+
+Pre-requisites
+==============
+
+- Odoo 16 database
+- :abbr:`IoT (Internet of Things)` compatible devices
+- Device drivers for Windows.
+- Windows computer (laptop, desktop, server)
```

Comment:
might be worth to precise which OS version we support (Windows 10 and > ?). I already foreseen some Windows XP customers complaining that it does not works (or 32bits/64 bits issues). Might be worth to check which Windows versions are really compatible

Reply:
Noted, I added that Odoo recommends an updated version of Windows as older versions can cause issues. This is a good point, thanks for adding this note.